ui/docs/reference/core/worker-manager/worker-interaction.mdx

```diff
@@ -58,6 +58,19 @@ The expiration time of the new credentials comes from the worker pool configurat
 which gets set when calling [`workerManager.registerWorker`](/docs/reference/core/worker-manager/api#registerWorker).
 The default behavior, when `reregistrationTimeout` is omitted, is to extend the credentials by 96 hours.
 
+## Claim Work Timeout
+
+To make sure that spawned workers are not stuck in a "zombie" state, the worker-manager will check with queue service
```

Comment:
This would be a great place to explain what a zombie state is.

Reply:
great! as soon as we come up with final parameter name for this one :)

Reply:
done